src/content-script/index.mjs

```diff
@@ -32,11 +40,16 @@ async function run(question) {
   port.postMessage({ question });
 }
 
-const searchInput = document.getElementsByName("q")[0];
+const siteNameReplaceList = [document.location.hostname, "www.", "search.", "cn."]
+const siteName = siteNameReplaceList.reduce((pre, cur) => {
+  return pre.replace(cur, "")
+}).match(/(.+?)\./)[1]
```

Comment:
This is hard to read and understand

Reply:
replace the prefix of hostname, and then get first word before the dot with regex

do you think it would be better to change the implement, or just add some comments

Reply:
I think we can create a regex with Object.keys(config) and use it to match location.host